src/models/Campaign.py

```diff
@@ -45,29 +45,33 @@ def update(self, data):
       setattr(self, key, value)
     self.updated_at = datetime.datetime.utcnow()
     db.session.commit()
-  
+
   def delete(self):
     db.session.delete(self)
     db.session.commit()
-  
-  @staticmethod 
```

Comment:
did you use a linter or a rubocop-esque tool to get rid of all these extra spaces? or are you just that good?